packages/web/src/scss/components/SkipLink/README.md

```diff
@@ -0,0 +1,32 @@
+# SkipLink
+
+The `SkipLink` component is used to provide a way for users to skip directly to the e.g. main content of a page,
+improving accessibility for keyboard and screen reader users.
+
+## Usage
+
+```html
+<body>
+  <nav aria-label="Navigation with skip links">
+    <a href="#main-content" class="SkipLinkItem"> Skip to main content </a>
+  </nav>
+  <!-- Other navigation elements -->
+  <main id="main-content">
+    <!-- Main content -->
+  </main>
+</body>
+```
+
+⚠️ Ensure that the `SkipLink` is placed before the section you want to skip to.
```

Comment:
I would prefer to mention the classic example. E.g., it is the first item on the page and links to the main section id.

Reply:
this sounds like you should place it right before the section you link to.